<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
<author>David Sugar</author>
 <comment>** Copyright (C) 2017 Tycho Softworks.
**
** This program is free software: you can redistribute it and/or modify
** it under the terms of the GNU General Public License as published
** by the Free Software Foundation, either version 3 of the License, or
** (at your option) any later version.
**
** This program is distributed in the hope that it will be useful,
** but WITHOUT ANY WARRANTY; without even the implied warranty of
** MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
** GNU General Public License for more details.
**
** You should have received a copy of the GNU General Public License
** along with this program.  If not, see &lt;http://www.gnu.org/licenses/&gt;.</comment>
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QStackedWidget" name="pagerStack"/>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionLogin"/>
   <addaction name="actionSessions"/>
   <addaction name="actionContacts"/>
   <addaction name="actionOptions"/>
  </widget>
  <action name="actionSessions">
   <property name="text">
    <string>Activity</string>
   </property>
   <property name="toolTip">
    <string>Call and message activity</string>
   </property>
  </action>
  <action name="actionOptions">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionContacts">
   <property name="text">
    <string>Contacts</string>
   </property>
   <property name="toolTip">
    <string>Personal Contacts</string>
   </property>
  </action>
  <action name="actionLogin">
   <property name="text">
    <string>Login</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
